1. values added:  20
2. final result:  20
3. values added:  20
4. An error because the variable result has block scope so iot cannot be sccessed outside of the block. As line 13 is outside of the block, it cannot access the variable result so an error (ReferenceError: result is not defined) happens
5. AN error results because on line 7 the code attempts to reassign the value of result. Becasue result is initialzied wiht teh const kerword, whihc prevents this, the code returs wiht an error before lines 9 and 13 can print anything. 
6. AN error results because on line 7 the code attempts to reassign the value of result. Becasue result is initialzied wiht teh const kerword, whihc prevents this, the code returs wiht an error before lines 9 and 13 can print anything. 

